================================================================================
 v1.3.1 (to be released)
================================================================================
Fixed: ic() invocations that fail to find or access source code (e.g. eval(),
  exec(), python -i, etc) now print an error message instead of throwing an
  IOError (Python 2) or OSError (Python 3).


================================================================================
 v1.3
================================================================================
First release.

This changelog wasn't maintained prior to v1.3.
